Makefile

```diff
@@ -83,7 +83,7 @@ mypy:
 check-safety:
 	poetry check
 	# TODO remove numpy ignore flag when fixed
-	poetry run safety check --full-report -i 44715
+	poetry run safety check --full-report -i 44715 -i 51668
```

Comment:
is this an automated change?

Reply:
This is to ignore a security vulnerability check unrelated to my changes - cleared with @ddl-gabrielhaim